aapcs64/aapcs64.rst

```diff
@@ -960,6 +961,15 @@ the subroutine must ensure that p4-p15 are preserved across the
 call. In other cases it need not preserve any scalable predicate
 register contents.
 
+First Fault Register (FFR)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The special-purpose First Fault Register (FFR) has the same size and
+format as the scalable predicate registers, and is available if and only
```

Comment:
`...and is available if and only they are.`: I think you dropped an if?

I'm guessing this should read: `..and is available if and only if they are.`

I would personally prefer: `..and is only available if they are.`, to make the sentence a bit more clear/readable.

Reply:
Ah, I did drop an if indeed, thanks for spotting that! I was trying to make this consistent with the section on Scalable Predicate Registers above, which says:
> These registers are available if and only if the scalable vector registers are available
Is it OK if I add the missing "if" to keep this consistent with the above, or would you still prefer a re-word?

Reply:
Thanks, I've reworded this as suggested in the latest version.